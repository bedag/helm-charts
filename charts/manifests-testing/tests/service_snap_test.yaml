suite: Service Snapshot Test
templates:
  - manifests.yaml
tests:
  - it: Full Configured (Snapshot)
    values:
      - ./values/manifests/service.yaml
    asserts:
      - matchSnapshot: {}