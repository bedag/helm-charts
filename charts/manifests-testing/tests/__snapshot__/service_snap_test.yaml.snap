Full Configured (Snapshot):
  1: |
    apiVersion: v1beta1
    kind: Service
    metadata:
      annotations:
        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: 443,8443
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: RELEASE-NAME-manifests-testing
        app.kubernetes.io/version: 0.0.0
        test.label/1: true
      name: RELEASE-NAME-manifests-testing
      namespace: custom-namespace
    spec:
      ports:
      - name: http
        nodePort: null
        port: 8888
        protocol: TCP
        targetPort: http
      - name: discovery
        nodePort: null
        port: 10000
        protocol: TCP
        targetPort: broadcast
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: RELEASE-NAME-manifests-testing
      type: ClusterIP
